cli/sql_util.go

```diff
@@ -279,12 +290,21 @@ func formatVal(val driver.Value) string {
 	case nil:
 		return "NULL"
 	case []byte:
-		// We don't escape strings that contain only printable ASCII characters.
-		if len(bytes.TrimLeftFunc(t, func(r rune) bool { return r >= 0x20 && r < 0x80 })) == 0 {
-			return string(t)
+		// We don't escape valid strings that contain only printable characters.
```

Comment:
btw we should merge https://github.com/lib/pq/pull/436 into cockroachdb/pq and then all this crap can go away